src/styles/_variables.scss

```diff
@@ -1,19 +1,5 @@
-//fonts
-$base-font: Montserrat;
-$size: 14px;
-$forum-font: Forum;
-
-//colors
-$orange: #ffb522;
-$white: #ffffff;
-$ocher: #4f3606;
-$black: rgb(0, 0, 0);
-$violet: #8649ef;
-//shadow
 $base-box-shadow: 0px 4px 28px rgba(0, 0, 0, 0.1);
 $border-routine-table: 0.5px solid #dedede;
 
-//border
-
 //border-menu
 $border-menu: 0.5px solid #dedede;
```

Comment:
@MaxOvs19 Also, if this style uses by one component - don't need to move it in the file. Same as for "$border-routine-table". And think about name "border-routine-table". Is it correct?)